web/forms.py

```
@@ -1654,3 +1662,159 @@ class Meta:
+
+    class Meta:
+        model = SelfCheckin
+        fields = ["post_rating", "notes"]
+
+
+class SessionFeedbackForm(forms.ModelForm):
+    """Form for submitting Rose, Bud, Thorn feedback for a session"""
+
+    rose = forms.CharField(
+        widget=TailwindTextarea(
+            attrs={"rows": 3, "placeholder": "Something that went well or that you liked about the session"}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)